README.md

```diff
@@ -1,53 +1,113 @@
-# {{ NAME }}
+# TimeSpan
 
-{{ DESCRIPTION }}
+A PowerShell module for formatting TimeSpan objects into human-readable strings with customizable precision and formatting options.
 
 ## Prerequisites
 
-This uses the following external resources:
-- The [PSModule framework](https://github.com/PSModule) for building, testing and publishing the module.
+This module uses the [PSModule framework](https://github.com/PSModule) for building, testing and publishing the module.
 
 ## Installation
 
-To install the module from the PowerShell Gallery, you can use the following command:
+To install the module from the PowerShell Gallery, you can use the following commands:
 
+### For PowerShell 5.1 and later
 ```powershell
-Install-PSResource -Name {{ NAME }}
-Import-Module -Name {{ NAME }}
+Install-Module -Name TimeSpan -Repository PSGallery
+Import-Module -Name TimeSpan
+```
+
+### For PowerShell 7+ with PSResourceGet
+```powershell
+Install-PSResource -Name TimeSpan
+Import-Module -Name TimeSpan
+```
+
+### For the latest version
+```powershell
+Install-Module -Name TimeSpan -Force
 ```
 
 ## Usage
 
-Here is a list of example that are typical use cases for the module.
+The primary function in this module is `Format-TimeSpan`, which converts TimeSpan objects into human-readable formatted strings.
```

Comment:
Lets remove this from this place, but integrate this language to the example text between the header and the script.